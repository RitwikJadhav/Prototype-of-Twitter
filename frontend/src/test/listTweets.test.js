import React from 'react';
import Enzyme, { mount, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import ListTweets from '.././components/lists-module/listTweets';
import renderer from 'react-test-renderer';

Enzyme.configure({ adpater: new Adapter() });

it('should render List Tweets Page correctly', () => {
    const component = renderer.create(<ListTweets />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});