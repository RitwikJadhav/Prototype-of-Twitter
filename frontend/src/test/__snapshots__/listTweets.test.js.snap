// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render List Tweets Page correctly 1`] = `null`;
